enterprise/app/esbuild.config.mjs

```diff
@@ -1,15 +1,25 @@
 import * as path from "path";
+import { stat } from "node:fs";
 
 // browserifyPathPlugin remaps the nodejs "path" module to "path-browserify".
 // This is needed for libsodium: https://github.com/evanw/esbuild/issues/1786
 let browserifyPathPlugin = {
   name: "path-browserify",
   setup(build) {
+    // detect if we are building from buildbuddy-internal repo and
+    // adjust 'node_modules' path accordingly
+    let entry = path.join(process.cwd(), "node_modules");
+    stat(entry, (err, _) => {
+      if (err) {
+        entry = path.join(process.cwd(), "external/com_github_buildbuddy_io_buildbuddy/node_modules");
+      }
+    });
```

Comment:
Instead of `stat`, you can use `fs.exists`. Also, I think we should use the `sync` API (`existsSync`) since there might be a race condition if the `exists` call isn't completed by the time the `onResolve` handler is invoked.

Reply:
`existsSync` is much cleaner. Good call!